### PR TITLE
fix(tls) exposes native canonicalizeIP and fix rootCertificates

### PR DESCRIPTION
**Pull request by cirospaciari**
Depends on https://github.com/Jarred-Sumner/uSockets/pull/10 and https://github.com/Jarred-Sumner/uWebSockets/pull/20

### CODE REVIEW

File: src/bun.js/bindings/ZigGlobalObject.cpp [hunk 1]
```diff
@@ -127,6 +127,7 @@
 using namespace Bun;
 
 extern "C" JSC::EncodedJSValue Bun__fetch(JSC::JSGlobalObject* lexicalGlobalObject, JSC::CallFrame* callFrame);
+extern "C" JSC::EncodedJSValue BUN__canonicalizeIP(JSC::JSGlobalObject* lexicalGlobalObject, JSC::CallFrame* callFrame);
```
Comment (Jarred-Sumner) [inline src/bun.js/bindings/ZigGlobalObject.cpp]: Can we do Bun case just to match convention

**Review by Jarred-Sumner — APPROVED** [review]

**Comment by Jarred-Sumner** [comment]
uWebSockets PR says there’s a conflict

**Comment by cirospaciari** [comment]
> uWebSockets PR says there’s a conflict

solved

**Comment by Jarred-Sumner** [comment]
Thank you

**Pull request merged**